aquamqtt.yaml

```
@@ -397,9 +397,18 @@ mqtt:
         manufacturer: tspopp
         name: AquaMQTT 
         model: V1
+  
+  water_heater:
+    - name: "AquaMQTT boiler"
+      icon: mdi:water-boiler
+      temperature_state_topic: "aquamqtt/hmi/waterTempTarget"
+      temperature_command_topic: "aquamqtt/ctrl/waterTempTarget"
+      current_temperature_topic: "aquamqtt/main/waterTemp"
+      unique_id: atlantic_water_boiler_thermostat
+      precision: 0.5
+   

```

Let's use your latest update before merging :)

```suggestion
  
  water_heater:
    - name: "AquaMQTT boiler"
      icon: mdi:water-boiler
      modes:
      - "ABSENCE"
      - "AUTO"
      - "BOOST"
      - "MAN ECO OFF"
      - "MAN ECO ON"
      mode_state_topic: "aquamqtt/hmi/operationMode"
      mode_command_topic: "aquamqtt/ctrl/operationMode"
      temperature_state_topic: "aquamqtt/hmi/waterTempTarget"
      temperature_command_topic: "aquamqtt/ctrl/waterTempTarget"
      current_temperature_topic: "aquamqtt/main/waterTemp"
      unique_id: atlantic_water_boiler_thermostat
      precision: 1.0
```